### ISSUE TITLE
Stuck on Steam Menu 

### ISSUE THREAD

**Issue by RKNINE** [polygraphene/ALVR#547]
Using ALVR on Oculus Quest 
I have the following issue:

When loading the ALVR application it says "Press the Menu button to begin"
however only one controller shows at a time and I am unable to press any buttons, it doesn´t show an oculus controller it shows the Vive controller.
 I have attempted to see if there is a relevant update through SideQuest,, however the following error then shows: INSTALL_FAILED_UPDATE_INCOMPATIBLE: Package com.polygraphene.alvr signatures do not match the previously installed version; ignoring!

If anyone can point me in the right direction that would be great!

**Comment by jellosubmarine** [comment]
I am experiencing the same problem. Latest build, v5 alpha.

**Comment by Rennan2468** [comment]
I have the same problem but with Gear VR, the Gear VR controller doesn't have a menu button so I'm stuck on the press menu button to begin screen.

**Comment by Jimmy5k** [comment]
This seems to link with a problem in 3 other posts.
ALVR on Oculus Quest:
Controls are out of whack.

I had to"In SteamVR, go to Settings > Reset, and click Remove all SteamVR USB Devices. Click Yes. Quit SteamVR. Reboot".

Also to the OP, i think it sounds like the server version and the package on your headset might be different.

I used the android powershell command prompt to install the package (make sure both package and server are same version). I didn't use sidequest so it might be forcing a different version to what you're using.

I hope this helps the OP.

As for Rennan: i found that using both x and y on the left controller opened the menu. rebinding of controls is required... but this is still something i'm trying to nut out myself.
